packages/app/src/utils/calculate-distance.ts

```diff
@@ -0,0 +1,12 @@
+import { TimestampedValue } from '@corona-dashboard/common';
+import { Point } from '@visx/point';
+import { HoverPoint } from '~/components-styled/line-chart/components';
+
+export function calculateDistance(
+  point1: HoverPoint<TimestampedValue>,
+  point2: Point
+) {
```

Comment:
the api of this util looks weird, with two different point types. Does something like this work?
```suggestion
export function calculateDistance<T extends { x: number; y: number }>(
  point1: T,
  point2: T
) {
```